spec/support/vcr_cassettes/vass/cancel_appointment_success.yml

```
@@ -1,5 +1,25 @@
 ---
 http_interactions:
+- request:
+    method: post
+    uri: https://login.microsoftonline.us/test-tenant-id/oauth2/v2.0/token
```

You can use `VASS_AUTH_URL` here (since you have it defined in vrc.rb).
```suggestion
    uri: <VASS_AUTH_URL>/test-tenant-id/oauth2/v2.0/token
```

Please update the other cassettes in this PR to do the same.